docs/quick_start.md

```diff
@@ -180,19 +159,19 @@ Note: *If any changes are made to the dependencies section of the `package.json`
     "test": "react-scripts test",
     "prebuild": "electron-rebuild",
     "build": "react-scripts build",
-    ...    
+    ...
 }
 ```
 
 The scripts section contains a number of commands you can run.
 
 They are also presented in key/value pair where the key is the command name and value is the command we want to run.
 
-*Checkout the [NPM Scripts](https://brown-ccv.github.io/honeycomb-docs/docs/npm_scripts) page for more information about the different scripts you can run.*
+_Checkout the [NPM Scripts](https://brown-ccv.github.io/honeycomb-docs/docs/npm_scripts) page for more information about the different scripts you can run._
 
 ### 4. Run the task in dev mode
 
-*Windows users must use a git bash terminal*
+Note: **Windows users must use a git bash terminal**
```

Comment:
Does everyone know what a git bash terminal is? I feel like when I told this to people some where confused

Reply:
I could include a screenshot?

Reply:
Maybe adding the instruction of opening the File explorer and searching git bash terminal to find it will help?

Reply:
Oh nvm, screenshot looks great